WooCommerce/Classes/ViewRelated/Dashboard/DashboardViewModel.swift

```diff
@@ -248,6 +256,15 @@ final class DashboardViewModel: ObservableObject {
             .store(in: &subscriptions)
     }
 
+    func showCustomizationScreen() {
+        if shouldSaveCurrentCards {
```

Comment:
Nit: we can simplify the logic by reusing the `showNewCardsNotice` check and removing the extra property `shouldSaveCurrentCards`.

Reply:
Ah good eye. This variable was added before I added loading saved cards in `configureNewCardsNotice()`, so this was necessary. Not anymore! Updated in https://github.com/woocommerce/woocommerce-ios/pull/12861/commits/9a7b799de0545c1bd59dbd40eaaca960a0101471